src/Firebase.utils.js

```diff
@@ -1,6 +1,8 @@
 // import firebase from "firebase";
+// import dotenv from 'dotenv'
 import firebase from "firebase/compat/app";
 import "firebase/compat/firestore";
+// dotenv.config()
 
 var firebaseConfig = {
     apiKey: "AIzaSyDofdnlbXgxhPja0PHR7ZF6DV482Uy8KXM",
```

Comment:
what is the use of .env file if your are using values directly? Make sure you use env values @Kunal-inno